websites/O/OpenFront.io/presence.ts

```diff
@@ -0,0 +1,72 @@
+import { Assets } from 'premid'
+
+const presence = new Presence({
+  clientId: '1395486313251213494',
+})
+
+enum ActivityAssets {
+  Logo = 'https://i.imgur.com/Lc2MCM6.png',
+}
+
+let isInGame = false
+let gameStartTimestamp: number | null = null
+
+presence.on('UpdateData', async () => {
+  const path = window.location.pathname.toLowerCase()
```

Comment:
For consistency with other Activities
```suggestion
  const path = document.location.pathname.toLowerCase()
```